keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepository.kt

```diff
@@ -41,51 +41,51 @@ interface ResourceRepository {
   /**
    * Invokes [callback] once with each registered resource.
    */
-  fun allResources(callback: (ResourceHeader) -> Unit)
+  suspend fun allResources(callback: (ResourceHeader) -> Unit)
```

Comment:
I really like the fact that you need to be explicit about blocking when using `ResourceRepository` now. Forces you to think about handling IO.